### ISSUE TITLE
[BUG] Not sure what this means 

### ISSUE THREAD

**Issue by skyisthela** [da-rth/yasb#136]
## Expected Behavior
commands run and the bar shows up

## Current Behavior
command is unable to run 

## Steps to Reproduce

1. install python and dependencies
2. run python src/main.py
3. this shows up 
![image](https://github.com/da-rth/yasb/assets/150245867/1d4da908-156c-4a1a-b64e-d85bb326b26f)

**Comment by azinsharaf** [comment]
have you changed directory to the repo's directory you have cloned?

**Comment by skyisthela** [comment]
> have you changed directory to the repo's directory you have cloned?

i don't think so, let me try it now

**Comment by skyisthela** [comment]
> have you changed directory to the repo's directory you have cloned?

yep, same issue. i typed cd and dragged the yasb folder, ran python src/main.py and the same issue persisted

**Comment by hash4q** [comment]
just open the the terminal in src folder and run py main.py

**Comment by skyisthela** [comment]
> just open the the terminal in src folder and run py main.py

there's no main.py found. and by src folder do you mean the yasb folder or?

**Comment by skyisthela** [comment]
> just open the the terminal in src folder and run py main.py

i cloned the repo (a bit weird this wasn't mentioned) and ran said command in the cmd, this shows up instead, i assume this is an issue with one of the requirements?
![image](https://github.com/da-rth/yasb/assets/150245867/8be0a6c2-ac31-44ce-809c-7d6956cd10e4)
